### ISSUE TITLE
support for terraform 0.15

### ISSUE THREAD

**Issue by capdragon** [jason-johnson/azure-pipelines-tasks-terraform#44]
Terraform has [removed some arguments](https://www.terraform.io/upgrade-guides/0-15.html#azure-backend-removed-arguments) in their 0.15 release.

Can we add these new arguments to support 0.15?

Old Name | New Name
-- | --
arm_client_id | client_id
arm_client_secret | client_secret
arm_subscription_id | subscription_id
arm_tenant_id | tenant_id

**Comment by charleszipp** [comment]
This has already been done. If the task detects terraform 0.15 or higher is being used, it will switch to using those arguments automatically

**Comment by capdragon** [comment]
Thanks @charleszipp. So how can I upgrade to the latest version of the extension?

I tried uninstalling/reinstalling extension from marketplace because I noticed you updated yesterday.
![image](https://user-images.githubusercontent.com/917643/119671187-30ebc500-bdff-11eb-82ee-666b8550d0fa.png)

But still, looks like I'm running an older version because I still get errors related to new parameters in 0.15 (arm_client_id, etc) and my extension does not look like yours:
Yours:
![image](https://user-images.githubusercontent.com/917643/119671027-069a0780-bdff-11eb-8241-ce91be9483da.png)

Mine:
![image](https://user-images.githubusercontent.com/917643/119671644-8cb64e00-bdff-11eb-8c70-e813cdb15451.png)
